lighthouse-core/test/scripts/i18n/bake-ctc-to-lhl-test.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-env jest */
+
+const bakery = require('../../../scripts/i18n/bake-ctc-to-lhl.js');
+
+describe('Baking Placeholders', () => {
+  it('passthroughs a basic message unchanged', () => {
+    const strings = {
+      hello: {
+        message: 'world',
+      },
+    };
+    const res = bakery.bakePlaceholders(strings);
+    expect(res).toEqual({
+      hello: {
+        message: 'world',
+      },
+    });
+  });
+
+  it('bakes a placeholder into the output string', () => {
+    const strings = {
+      hello: {
+        message: '$MARKDOWN_SNIPPET_0$',
+        placeholders: {
+          MARKDOWN_SNIPPET_0: {
+            content: '`World`',
+          },
+        },
+      },
+    };
+    const res = bakery.bakePlaceholders(strings);
+    expect(res).toEqual({
+      hello: {
+        message: '`World`',
+        placeholders: undefined,
+      },
+    });
+  });
+
+  it('throws when a placeholder cannot be found', () => {
+    const strings = {
+      hello: {
+        message: 'Hello $MARKDOWN_SNIPPET_0$ $MARKDOWN_SNIPPET_1$',
+      },
```

Comment:
maybe do a placeholder for `MARKDOWN_SNIPPET_1` but not 0 for this test? Or add a case for that if you want to keep the no placeholders case?